internal/ee/infrastructure/helm/docs/OBSERVABILITY-ADOT.md

```
@@ -0,0 +1,197 @@
+  metrics:
+    enabled: true
+    endpoint: myrelease-flexprice-adot-collector.flexprice.svc:4317
+  logs:
+    enabled: true
+    endpoint: myrelease-flexprice-adot-collector.flexprice.svc:4317
+```
+
+Replace `<release>`, `<ns>` with your actual Helm release name and namespace.
+In the example above, the release name is `myrelease` and the namespace is
+`flexprice`. The endpoint format is:
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)